USAGE_DATA.md

```diff
@@ -23,15 +27,27 @@ Note that these commands do not include `--help` commands. We do not collect dat
 Alternatively you can _disable_ telemetry by setting `ODO_DISABLE_TELEMETRY` environment variable to `true`.
 This environment variable will override the `ConsentTelemetry` value set by `odo preference`.
 
-The following table describes the additional information collected by odo commands.
+The following tables describes the additional information collected by odo commands.
```

Comment:
Can you add this under a dedicated section? By reading the rendered output, I had the feeling that this was part of the "Enable/Disable preference > Disable" section.

Suggestion:
```suggestion
### Additional information collected

The following tables describe the additional information collected by `odo` commands.
```

Reply:
Or you could also make "Enable/Disable preference" the last section.